### PR TITLE
Develop react_movies-list-filter

### PR DESCRIPTION
**Pull request by sergey-lipikhin**
[DEMO LINK](https://sergey-lipikhin.github.io/react_movies-list-filter/)

### CODE REVIEW

**Review by Anton-Tretiak — APPROVED** [review]
Good job!

File: src/App.tsx [hunk 1]
```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+function hasContainQuery(searchWithin: string, query: string): boolean {
+  return searchWithin.toLowerCase().includes(query.trim().toLowerCase());
```
Comment (OleksiiBlezniuk) [inline src/App.tsx:7]: This line is too long, let's move result of `lowerCase` to constant

File: src/App.tsx [hunk 2]
```diff
@@ -20,12 +30,14 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                value={query}
+                onChange={(event) => setQuery(event.target.value)}
```
Comment (OleksiiBlezniuk) [inline src/App.tsx:34]: Let's create separate handler function for it

**Review by sergii-nosachenko — APPROVED** [review]
Good work 👍

**Pull request closed without merging**